### DISCUSSION TITLE
How do I skip a local Plex GUID ?

### DISCUSSION THREAD

**Discussion by gitttt54** [meisnate12/Plex-Meta-Manager#1049] [Q&A]
Hello,

I got 1000s of entries in he log like: 

`[2022-08-12 10:03:30,621] [tmdb.py:403]               [INFO]     | MOVIENAME | No TMDb ID for Guid: local://926052                                                                                                                                        |`

This error is a true error, because all of those movies have NO existing TMDb (or TVDb or IMDb or any other Db at all) ID and never will.

Those are all personal videos/movies/documentations/presentations/shorts/lectures/talks/etc being legworked into the plex libraries.

_And so, there is no need to handle them on every run._

They are spread all over the libraries, so I can not just exclude those libraries.

Therefore my question:

How do I exclude a given numbers of local plex GUIDS ?

I only found "Ignore IDs[¶](https://metamanager.wiki/en/latest/config/settings.html?highlight=ids#ignore-ids)" and Ignore IMDb IDs[¶](https://metamanager.wiki/en/latest/config/settings.html?highlight=ids#ignore-imdb-ids) in the wiki.

Thank you very much!

**Comment by meisnate12** [comment]
whats the agent youre using if its the personal media agent then it should ignore these messages

**Comment by gitttt54** [comment]
Edit Library shows this:
<img width="402" alt="1" src="https://user-images.githubusercontent.com/96403800/192509874-644ef858-0356-4a0e-b807-a1a1c1fc60e6.png">
So I guess, the library uses those settings:
<img width="1051" alt="2" src="https://user-images.githubusercontent.com/96403800/192510016-e74befe0-fd4a-400b-8122-e741ef38dc34.png">
But actually, I don't know, I have this too under plexweb->settings->agents
<img width="1055" alt="3" src="https://user-images.githubusercontent.com/96403800/192510132-3c59954a-2375-4d24-9606-588ace1c2fce.png">
